### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by bricejar** [tbolis/react-sketch#74]
Hi, when do you plan of releasing a new version?

**Comment by schlaegerz** [comment]
Yeah this would be helpful as I have been waiting to be able to try this package until the dependencies bug has been figured out. This package looks really great and I would love to be able to use it.

**Comment by imperio59** [comment]
+1 to this, can we get a new release please?

**Comment by bricejar** [comment]
A lot of bugs have been fixed in the last version of fabricjs. It seems that the authour has abandones his package. 

You can always compile the package yourself with fabricjs 3.3.2 :)

**Comment by bogdancochioras** [comment]
> A lot of bugs have been fixed in the last version of fabricjs. It seems that the author has abandoned his package.
> 
> You can always compile the package yourself with fabricjs 3.3.2 :)

Hello, can you give some steps on how to do that?
